nuto/math/EigenSparseSolve.cpp

```diff
@@ -20,6 +22,7 @@ namespace NuTo
 template <typename TSolver>
 Eigen::VectorXd SolveWithSolver(const Eigen::SparseMatrix<double>& A, const Eigen::VectorXd& b)
 {
+    Timer t(__FUNCTION__, true, Log::Debug);
```

Comment:
Why is the timer class inside the solver, shouldn't that be outside? Where is it used?